app/invocation/invocation.css

```diff
@@ -530,6 +530,41 @@
   margin-right: 10px;
 }
 
+#cache-miss-popup {
+  right: auto;
+  flex-direction: column;
+  padding: 12px;
+}
+
+#cache-miss-popup .title {
+  font-weight: 600;
+  font-size: 15px;
+  background: white;
+  padding: 4px;
+}
+
+#cache-miss-popup button {
+  margin-top: 5px;
+}
```

Comment:
nit, use a class rather than ID here? not a big deal but we don't usually use IDs for stuff

Reply:
For now, the way that we increase specificity is by either making the selector more deeply nested, or adding additional classnames.

e.g.

```
/* more deeply nested */
.cache-miss-popup .button.blah { ... }

/* add more classnames */
.cache-miss-popup .title.title { ... } /* duplicate .title selector for higher specificity */
```

In the long run, I think we need a better system, but this what we've been using so far :shrug:

Reply:
I am having trouble getting the CSS to apply to the popup itself (not the children elements). Just going to stick with the ID for now. Personally find it a bit more readable than the duplicated class name

Reply:
You should be able to write `.popup.cache-miss-popup` to get higher specificity than any of the `.popup` rules in `popup.css`. If that still doesn't work let me know and I can try to restructure our CSS setup to make it more friendly to overriding.

The reason I am trying to avoid IDs here is because it's introducing a "new way" to override specificity that we aren't doing anywhere else in the app, so it's adding a new thing that someone approaching the codebase would have to learn. The fewer conventions we are using throughout the app, the better.

Reply:
Ahhh thank you - TIL how to use no space to target multiple class names on the same element. That fixed things for me. Updated

Reply:
np, thanks for taking the time to fix that :)